lib/category_list/apis/category_api.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:dio/dio.dart';
+import 'package:flex_storefront/category_list/models/category.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart';
+
+const DOMAIN = 'https://flex-cms-fpnplvnjqq-uc.a.run.app';
+const PATH = '/api/categories';
+
+class CategoryApi {
+  final http = Dio()
+    ..options.headers = {
+      'Authorization': 'Bearer ${dotenv.get('STRAPI_TOKEN')}',
+    };
+
+  Future<List<Category>> fetchCategories({int? parentId}) async {
```

Comment:
Will parentId always be an `int` or would this be better served as a `String` in most cases?

Reply:
If we move out of Strapi it possibly won't, but let's deal with it when the time will come (if it comes).